packages/external-integration/e2e/ExternalPaymentMethod.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    PaymentStepAsGuestPreset,
+    test,
+} from '@bigcommerce/checkout/payment-integration-test-framework';
+
+import { internalOrder, order, orderPayment } from './MockResponse';
+
+test.describe('External payment method', () => {
+    test('Sezzle is working', async ({ assertions, checkout, page }) => {
+        // Testing environment setup
+        const responseProps = { status: 200, contentType: 'application/json' };
+
+        await checkout.use(new PaymentStepAsGuestPreset());
+        await checkout.start('Sezzle');
+        await checkout.route(
+            'https://sandbox.checkout.sezzle.com?id=**',
```

Comment:
Should this be a part of `sezzle-integration` if it is testing Sezzle specifically?

Reply:
I agree. I can rename this package as `sezzle-integration` like what I did for the `offsite` payment method.